examples/Workflows/dog_breed_classification/README.md

```diff
@@ -0,0 +1,72 @@
+# Workflow pipeline example for classifying dog breed
+
+This workflow uses two models, first to classify if an image is of a dog or a cat. If it is a dog the second model does breed classification.
+To change the default batch size and batch delay the yaml file for the workflow can to be changed. This cannot currently be set via the REST API.
+
+## Flow
+
+```
+input -> pre_processing --> cat-dog-classification -> dog-breed-classification -> output
+                    \                                         /
+                     \_______________________________________/
+```
+
```

Comment:
@maaquib please add some explanation/clarification that in each model handler how user can access the output of different nodes(e.g pre-processing, cat-dog-classification output)/ adding note that each model handler had access to the nodes that are defined in the DAG.